rules_java_gapic/java_gapic_pkg.bzl

```diff
@@ -83,13 +94,14 @@ def _gapic_pkg_tar_impl(ctx):
         deps = " ".join(["'%s'" % d.path for d in deps]),
         package_dir_path = paths.package_dir_path,
         package_dir = paths.package_dir,
+        samples = " ".join(["'%s'" % s.path for s in samples]),
         pkg = ctx.outputs.pkg.path,
         tar_cd_suffix = paths.tar_cd_suffix,
         tar_prefix = paths.tar_prefix,
     )
 
     ctx.actions.run_shell(
-        inputs = deps,
+        inputs = deps + samples,
```

Comment:
What does this `run_shell` do? Do we need it? I see that pretty much all other changes in bazel files are net new, but this is modifying existing scripts so a little alarming to me.

Reply:
Yes, it runs the script above it for the _gapic_pkg_tar_impl. 'samples' is being used as input {samples}. This is where they get unpacked into the correct location in the tar.